src/panel/direction/DirectionForm.jsx

```diff
@@ -85,6 +86,7 @@ export default class DirectionForm extends React.Component {
           <DirectionInput
             isLoading={isLoading}
             value={destinationInputText}
+            point={destination}
```

Comment:
I think we should pass point as props from now, and use its properties inside `DirectionInput`. wdyt ?

Reply:
If I remember correctly, we passed flat properties and not full point objects because we needed to separate the current input string itself from the point data that may be attached to the field.
There is probably a best way to do that now, but let's not change everything in this PR, as it's one of the biggest source of bugs in the app :)

I have high hopes (maybe too high?) we can improve the whole `DirectionPanel`/`DirectionForm`/`DirectionInput` group when:
1. `<Suggest>` has been simplified further (with requires the top bar to be implemented in React)
2. we move these components to the functional form, which often highlights better ways to manage complex states.